CHARTER.md

```diff
@@ -0,0 +1,89 @@
+# cosmos.gl Charter
+
+## **Section 0: Guiding Principles**
+
+**Mission:** Provide the web development community with a high-performance framework for visualizing network graphs and scatter plots.
+
+**Vision:** Deliver fast, robust, open-source data visualization tools that empower interactive, scalable analysis in the browser.
+
+**Values:**
+
+- **Performance First:** Prioritize speed and efficiency in every feature and implementation.  
+- **Accessibility:** Ensure the API is intuitive and the tooling easy to adopt, lowering barriers for developers.  
+- **Community:** Foster open collaboration, welcoming contributions and feedback.  
+- **Transparency:** Maintain clear documentation, benchmarks, and decision-making processes.
+
+## **Section 1: Scope**
+
+cosmos.gl is a browser-native, GPU-accelerated force-directed graph layout and rendering engine designed to visualize and interact with massive, complex datasets at scale. By leveraging WebGL, it delivers fast simulations and real-time rendering of millions of nodes and edges directly in the browser. cosmos.gl bridges the gap between high-performance data visualization and interactive web-based research workflows, serving developers, researchers, and analysts. Its value lies in unlocking scalable, explainable graph exploration for AI, biotech, finance, and data science stakeholders.
+
+### **1.1: In-scope**
+
+- GPU-accelerated graph algorithms  
+- WebGL-based rendering of large-scale graph structures  
+- Browser-native integration with frontend tooling and workflows
+
+### **1.2: Out-of-Scope**
+
+- Full-fledged graph analytics  
+- Backend data processing and pipelines
+
+## **Section 2: Relationship with OpenJS Foundation CPC**
+
+Technical leadership of the cosmos.gl project is delegated to the cosmos.gl Technical Steering Committee (TSC) by the OpenJS Cross Project Council (CPC). The TSC is also responsible for deciding how to allocate earmarked funds it might receive through the Foundation. Amendments to this charter require approval from both the CPC, through its [decision-making process](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-9-decision-making), and the TSC.
```

Comment:
I believe it'd make more sense just saying the TSC is responsible for overseeing the project/being the top-level governance body of the project.

Reply:
Agreed, simplified this in the latest commit